src/lpython/semantics/python_comptime_eval.h

```diff
@@ -178,7 +178,7 @@ struct PythonIntrinsicProcedures {
         ASR::ttype_t* arg_type = ASRUtils::expr_type(arg);
         if (ASRUtils::is_character(*arg_type)) {
             char* c = ASR::down_cast<ASR::StringConstant_t>(arg)->m_s;
-            std::string s = toupper(std::string(c));
+            std::string s = (int (*)(int))std::toupper(std::string(c));
```

Comment:
Why is this needed? I would prefer not to write code like this, unless there is no other way.